message_sender/tests.py

```diff
@@ -968,47 +1014,6 @@ def test_create_inbound_data_with_concurrency_limiter(self):
         self.assertEqual(d.transport_type, "voice")
         self.assertEqual(d.helper_metadata, {"session_event": "close"})
 
-    @responses.activate
```

Comment:
Are these two tests we are removing covered by other tests? Looks like the only place where we call the inbound url with a channel_id.